src/prefect/tasks/great_expectations/checkpoints.py

```diff
@@ -0,0 +1,100 @@
+"""
+Great Expectations checkpoints are combinations of data source, expectation suite, and validation operators configuration that can be used to run Great Expectations actions. 
+Checkpoints are the preferred deployment of this configuration; you can read more about setting up checkpoints 
+[at the Great Expectation docs](https://docs.greatexpectations.io/en/latest/tutorials/getting_started/set_up_your_first_checkpoint.html#set-up-your-first-checkpoint).
+
+You can use these task library tasks to interact with your Great Expectations checkpoint from a Prefect flow.
+"""
+import prefect
+from prefect import Task
+from prefect.engine import signals
+from prefect.utilities.tasks import defaults_from_attrs
+
+try:
+    import great_expectations as ge
+except ImportError:
+    pass
```

Comment:
There's an opportunity to raise a warning / error that this module requires `pip install "prefect[ge]"`, this is my only real note for you!

Reply:
(because I dont think "ge" is documented anywhere else as the extra name)

Reply:
Fixed in 5ab324dc335b216e4815ac84a884901442220319, also switched to the style of some of the other ones which raise the importerror it in `__init__.py` instead